negotiation/message/contract-agreement-verification-message.json

```diff
@@ -3,22 +3,9 @@
   "@type": "dspace:ContractAgreementVerificationMessage",
   "dspace:providerPid": "urn:uuid:a343fcbf-99fc-4ce8-8e9b-148c97605aab",
   "dspace:consumerPid": "urn:uuid:32541fe6-c580-409e-85a8-8a9a32fbe833",
-  "cred:credentialSubject": {
-    "@type": "dspace:ContractAgreementMessage",
-    "dspace:hash": "abcdefghijk12345"
+  "dspace:referencedContractAgreementMessage": {
```

Comment:
I would shorten this to `dspace:agreementMessage`, `dspace:message`, `dspace:hashedMessage` or  similar. We are never using the term "reference" for actual references.